supplementary_style_guide/style_guidelines/links.adoc

```diff
@@ -2,8 +2,53 @@
 [[links]]
 = Links
 
+The following guidelines improve findability and search engine optimization.
+
+.Technical best practices
+
+* Use hyperlinks consistently.
+* Do not use non hyperlinks.
+* Do not use JavaScript links. 
+* Do not use Flash links.
+* Do not use links in Frames or iframes.
+* Default to followable, crawlable links. Do not use “nofollow” links unless absolutely necessary.
+* Include links only when necessary.
+
+.Link text best practices
+
+* Contextually describe what the user will find at the target location so that they can decide if they want to leave their current location. 
+* Avoid irrelevant link text.
+* You can use a sentence or sentence fragment as the link text.
+
+
+
 [[internal-links]]
 == Internal links
 
+Internal links are links within www.redhat.com.
+
+You can use one of the following example link structures for consistency.
+
+.Example AsciiDoc for internal links
+----
+For more information about <topic>, see xref:<link>[<link_text>].
+----
+
+To link to a separate user story:
+----
+For more information about <topic>, see link:<link>[<link_text>] in _<title_name>_.
+----
+
+
 [[external-links]]
 == External links
```

Comment:
In my experience, linking to external sites is an area of caution as it implies Red Hat is tacitly endorsing content that we don't actually have quality control over and that could change without us being aware (i.e. I could link to a tech article that's applicable to my product, and then later the site owner changes it to a blog about Yeti sightings). Is this nuance worth documenting in this section?

Reply:
I've included a warning, wdyt, @mportman12 ?

Reply:
@ndeevy Your edit is good but recommend incorporating in @bredamc's change as well.

Reply:
I know that you try not to duplicate IBM Style info in this doc, but is it advisable to reiterate the legal restriction that deep links into another company's website can bypass terms of use? (In practical terms, this was often interpreted as the ability to link to a "book" because that book would have all the legal info included in standard boilerplate. But we did not link directly to a topic within that book, we gave a general description of the topics you could use to navigate there.)